.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+files: ^app/
+exclude: (^app/core/migrations/|^app/tests/data/|^app/core/management/commands/data/)
```

Comment:
```suggestion
exclude: (^app/core/migrations/|/tests/data/|^app/core/management/commands/data/)
```
would that exclude all tests/data directories?

Reply:
I think so, I will test it!